modules/gltf/src/lib/extensions/EXT_mesh_features.ts

```diff
@@ -20,6 +22,13 @@ export async function decode(gltfData: {json: GLTF}, options: GLTFLoaderOptions)
   decodeExtMeshFeatures(scenegraph, options);
 }
 
+export function encode(gltfData: {json: GLTF}, options: GLTFWriterOptions): {json: GLTF} {
+  const scenegraph = new GLTFScenegraph(gltfData);
+  encodeExtMeshFeatures(scenegraph, options);
+  scenegraph.createBinaryChunk();
```

Comment:
Should we create the binary chunk in the calling function? Otherwise it feels like we can only encode one extension? (I am assuming we only create one binary chunk at the end?)

Reply:
Right. I corrected it.